Kwf/Assets/CommonJs/Provider.php

```diff
@@ -141,6 +142,16 @@ private function _parseDependencies($dependency)
                 }
                 $dep = $dir . '/'. $dep;
             }
+
+            if ($depBrowserAlternatives) {
+                $path = $dependency->getAbsolutePath($dep);
+                $path = str_replace('/' . $dependency->getType() . '/', './', $path);
```

Comment:
don't st_replace on the whole string, /type/ might exist not only a the beginning